services/web-server/src/main.js

```diff
@@ -123,14 +124,15 @@ const load = loader(
     httpServer: {
       requires: ['app', 'schema', 'context', 'monitor'],
       setup: async ({ app, schema, context, monitor }) => {
+        const httpServer = createServer(app);
         const server = new ApolloServer({
           schema,
           context,
           formatError,
           tracing: true,
+          plugins: [ApolloServerPluginDrainHttpServer({ httpServer })],
```

Comment:
Says this is _highly recommended_ https://github.com/apollographql/apollo-server/blob/apollo-server-core%403.6.4/docs/source/api/plugin/drain-http-server.mdx.